templates.go

```diff
@@ -48,7 +48,7 @@ var templates = map[string][]string{
 		"github.com/u-root/u-root/cmds/core/id",
 		"github.com/u-root/u-root/cmds/core/init",
 		"github.com/u-root/u-root/cmds/core/insmod",
-		"github.com/u-root/u-root/cmds/core/io",
+		//"github.com/u-root/u-root/cmds/core/io",
```

Comment:
This would remove io from the minimal template for everything, not only tinygo. Is that intended?

Reply:
thanks, I also un-commented which, another mistake on my part. The only diff in this file now is adding tinygoize.